### PR TITLE
Add process scheduling class

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: resolwe/flow/static/flow/processorSchema.json [hunk 1]
```diff
@@ -39,6 +39,11 @@
             "type": "string",
             "pattern": "^RAW|CACHED|TEMP$"
         },
+        "scheduling_class": {
+            "description": "Scheduling class",
+            "type": "string",
+            "enum": ["interactive", "batch"]
```
Comment (dblenkus) [inline resolwe/flow/static/flow/processorSchema.json]: Options in `persistence` are enforced with `"pattern": "^RAW|CACHED|TEMP$"`. Make this consistent.

File: resolwe/flow/managers/base.py [hunk 2]
```diff
@@ -148,6 +148,9 @@ def communicate(self, run_sync=False, verbosity=1):
                     if program is not None:
                         priority = 'normal'
                         if data.process.persistence == Process.PERSISTENCE_TEMP:
+                            # TODO: This should probably be removed.
```
Comment (dblenkus) [inline resolwe/flow/managers/base.py]: Yes this is only for backward compatibility.

File: resolwe/flow/static/flow/processorSchema.json (hunk 1 above)
Comment (mstajdohar) [inline resolwe/flow/static/flow/processorSchema.json]: I prefer enum. Still, follow Domen's advice and make it consistent.

Comment (kostko) [inline resolwe/flow/static/flow/processorSchema.json]: I've fixed the `persistence` schema to use `enum` instead.

**Pull request merged**